### PR TITLE
Create Orbit device for archive data retrieval

### PR DESCRIPTION
**Pull request by xresende**
Implementation of `Orbit` class to allow orbit retrieval from archiver.
see example script in 
https://gitter.im/lnls-fac/si_drift

### CODE REVIEW

File: siriuspy/siriuspy/clientconfigdb/pvsconfig.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 _TIMEOUT = 0.5
 
+# NOTE: this module has become deprecated
```
Comment (anacso17) [inline siriuspy/siriuspy/clientconfigdb/pvsconfig.py:12]: @xresende why?

Comment (xresende) [reply siriuspy/siriuspy/clientconfigdb/pvsconfig.py:12]: Oops, I searched PVsConfig in `siriuspy`, thinking that all ramp algorithms had been moved to this repository and forgot to search in `hla`. Now I see that the class is being used there. I'll remove the comment. Thanks!

Comment (xresende) [reply siriuspy/siriuspy/clientconfigdb/pvsconfig.py]: done.

**Comment by xresende** [comment]
- Renamed `Orbit` based on individual BPM data to `OrbitBPM`
- Created `OrbitSOFB` based on SOFB `SlowOrb` PVs

**Pull request merged**